test/e2e/apimachinery/aggregator.go

```diff
@@ -42,6 +43,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	samplev1alpha1 "k8s.io/sample-apiserver/pkg/apis/wardle/v1alpha1"
 
+	"fmt"
```

Comment:
In the first section is normal for built in packages.

Reply:
Need to find how to control that in my IDE. Fixed.